pylib/anki/errors.py

```diff
@@ -17,6 +18,14 @@ def __str__(self) -> str:
         return self._localized
 
 
+class DocumentedError(LocalizedError):
+    """A localized error described in the manual."""
+
+    def __init__(self, localized: str, help_page: Any) -> None:
```

Comment:
I didn't get the type annotation to work with `anki.collection.HelpPage.V` or something similar.